lib/gollum-lib/filter/tags.rb

```diff
@@ -278,6 +291,8 @@ def css_options_for_link(kind)
       'class="internal absent"'
     when :page_present
       'class="internal present"'
+    when :internal_anchor
+      'class="internal-anchor"'
```

Comment:
I had `internal anchor` first, but it turns out the `anchor` class is already in use and actually makes links invisible. :)

Reply:
To clarify, the class is not defined in the CSS at present, but this is to a) make sure we can add it in the future b) make the exception in `sanitization.rb`

Reply:
Wouldn't it be cleaner to add `internal anchor` CSS as a subclass of anchor?